client/src/api/services/saved-package.service.ts

```diff
@@ -0,0 +1,40 @@
+import { SavedPackage } from '@/models/saved-packages.model';
+import { axiosInstance } from '../config/axios-instance';
+import { PackageDocument } from '@/models/package.model';
+
+export const ROUTE_PREFIX = '/saved-packages';
+
+export const SavedPackageService = {
+    async getUsersSavedPackages(packageId?: string) {
+        try {
+            const { data } = await axiosInstance.get<{ _id: string; packages: PackageDocument[] }[]>(
```

Comment:
```suggestion
            const { data } = await axiosInstance.get<History[]>(
```
Aint it gets HistoryItems from db?